Rakefile

```diff
@@ -4,6 +4,7 @@
 require File.expand_path('../config/application', __FILE__)
 require 'rake'
 require 'ci/reporter/rake/minitest' if Rails.env.test?
+require 'ci/reporter/rake/rspec'
```

Comment:
@h-lame Sorry for that, amended in https://github.com/alphagov/publisher/pull/541